buku.py

```diff
@@ -2186,7 +2186,7 @@ def auto_import_from_browser(self):
             username = os.getlogin()
             GC_BM_DB_PATH = 'C:/Users/{}/AppData/Local/Google/Chrome/User Data/Default/Bookmarks'.format(username)
 
-            DEFAULT_FF_FOLDER = 'C:/Users/{}/AppData/Roaming/Mozilla/Firefox/Profiles'.format(username)
+            DEFAULT_FF_FOLDER = 'C:/Users/{}/AppData/Roaming/Mozilla/Firefox/'.format(username)
```

Comment:
This works for the ini file. The actual profiles are within the `Profiles` directory.

```
- Firefox
---- Profiles
-------- ...default...
-------- ...profile1...
---- profiles.ini
```

Reply:
Path variable in `profiles.ini` is `Profiles/profile.name.default` for windows.

Reply:
OK. I'll test.